examples/demo/src/routes/HomePage.tsx

```diff
@@ -24,8 +20,9 @@ export const HomePage: React.FC = () => {
   ] = useApplicationContext();
 
   return (
-    <Recommend recommendClient={recommendClient}>
+    <>
       <TrendingFacets
+        recommendClient={recommendClient}
```

Comment:
does this have a reason to move the client out of the provider? Can you not mix Recommend from regular RecommendReact with the experimental one? Is there a good error message if it's mixed?

Reply:
I just wanted to show 2 ways to use Recommend with the Context Provider (PDP) and without (HomePage).

Reply:
And yes if you're using the Recommend Context (like the PDP), you will need to import all components from `/experimental-personalization`

Reply:
So is there a clear error if you use the regular provider with the experimental components? I didn't see it in the files.

Reply:
I just added a note in the PR description (that will go also in the release note). Again this is something we can guide the beta clients through their implementation.